### ISSUE TITLE
Protoflux nodes with `Multi` variants should be expandable to them

### ISSUE THREAD

**Issue by Zandario** [Yellow-Dog-Man/Resonite-Issues#395]
### Is your feature request related to a problem? Please describe.

It's frustrating that say `ValueAdd<Float>` cannot convert to `ValueAddMulti<Float>` and back. 

This has also made things a smidge confusing for newer users. For example I have had cases where someone gets confused as to why mine can expand and theirs can't.

This is also confusing for old users, as migrated LogiX content were able to do this.

### Describe the solution you'd like

Just like LogiX, I would like a button on for example, the `ValueAdd<Float>` that turns into `ValueAddMulti<Float>` with an additional field (I say this as they both start with the same number of inputs)

### Example Image
I will say I personally don't like how distinct the differences between the visuals would be, but that's a different issue.
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/34761877/b40b55a6-bd84-4818-9521-f34d430c32f8)

### Describe alternatives you've considered

I cannot think of a good alternative that doesn't require too much overhead or doesn't confuse users, I apologize.

### Additional Context

Additionally it would be nice if the Multi Variants were included in the groups with their singulars, probably as a subcategory in it.

**Comment by ShrikeAlvaron** [comment]
I believe the visuals difference is because the basic non-multi versions always show in overview mode, so if this is done that could be changed so that even with just two inputs using the basic non-multi add or other such node, it would show the normal view too.  Or the opposite perhaps, though I really prefer non-overview myself |3

**Comment by Frooxius** [comment]
Yes this is something that I planned to do, but didn't get to it.
